%\documentstyle[11pt,a4]{article}
%\documentclass[a4paper]{article}
\documentclass[a4paper, 10pt]{article}
% Seems like it does not support 9pt and less. Anyways I should stick to 10pt.
%\documentclass[a4paper, 9pt]{article}
\topmargin-1.0cm

\usepackage{fancyhdr}
\usepackage{pagecounting}
\usepackage[dvips]{color}
\usepackage{hyperref}
\usepackage{enumitem}

%\setlist[itemize]{itemsep=0mm}
%\setlist[enumerate]{itemsep=0mm}
\setlist[enumerate]{nolistsep,leftmargin=*}
\setlist[itemize]{leftmargin=*}
% Color Information from - http://www-h.eng.cam.ac.uk/help/tpl/textprocessing/latex_advanced/node13.html

% NEW COMMAND
% marginsize{left}{right}{top}{bottom}:
%\marginsize{3cm}{2cm}{1cm}{1cm}
%\marginsize{0.85in}{0.85in}{0.625in}{0.625in}

\advance\oddsidemargin-0.65in
%\advance\evensidemargin-1.5cm
\textheight9.2in
\textwidth6.75in
\newcommand\bb[1]{\mbox{\em #1}}
\def\baselinestretch{1.05}
%\pagestyle{empty}

\newcommand{\hsp}{\hspace*{\parindent}}
\definecolor{gray}{rgb}{0.4,0.4,0.4}
%\definecolor{gray}{rgb}{1.0,1.0,1.0}


\begin{document}
\thispagestyle{fancy}
%\pagenumbering{gobble}
%\fancyhead[location]{text} 
% Leave Left and Right Header empty.
\lhead{}
\rhead{}
%\rhead{\thepage}
\renewcommand{\headrulewidth}{0pt} 
\renewcommand{\footrulewidth}{0pt} 
%%\fancyfoot[C]{\footnotesize \url{http://www.pitt.edu/\~zhh18/dev}} 

%\pagestyle{myheadings}
%\markboth{Sundar Iyer}{Sundar Iyer}

\pagestyle{fancy}
\lhead{\textcolor{gray}{\it Zhiguang Huo (Caleb)}}
%\rhead{\textcolor{gray}{\thepage/\totalpages{}}}
%\rhead{\thepage}
%\renewcommand{\headrulewidth}{0pt} 
%\renewcommand{\footrulewidth}{0pt} 
%\fancyfoot[C]{\footnotesize http://www.stanford.edu/$\sim$sundaes/application} 
%\ref{TotPages}

% This kind of makes 10pt to 9 pt.

%\vspace*{0.1cm}
\begin{center}
{\LARGE \bf RESEARCH STATEMENT}\\
\vspace*{0.1cm}
\end{center}


My research interest lies in both statistical \textbf{methodology} and \textbf{application} on \textbf{genomics and bioinformatics}.
Nowadays, large amount of genomic data are publicly available and 
integrating these datasets provides unprecedented opportunities to reveal disease mechanisms.
I have worked on \textbf{horizontal omics meta-analysis} (combine multiple cohorts of the same type of omics data) and 
\textbf{vertical omics integrative analysis} (combine multi-level omics data of the same patient cohort),
which will help increase statistical power, interpretability, reproducibility and understanding of disease mechanism.
In terms of statistical methodology,
\textbf{I am particularly interested in modeling and optimization for high-dimensional data, Bayesian methods, graphical models and statistical computing,}
which are capable of accommodating the high-dimensional nature of genomic data.
In terms of genomics and bioinformatics application,
I have collaborated with biologists in fields of \textbf{cancer} and \textbf{psychiatry} to analyze a broad range of genomic data (e.g. microarray and sequencing data),
which motivates me to develop practical methodology and user-friendly softwares.
However, genomics and bioinformatics are fast-evolving field and 
I am open minded to expand my research towards new data types (e.g. imaging data, \textbf{single cell data}) in the future.
My long term goal is to bridge bioinformatics and statistics/machine learning.

Below is a highlight of my \textbf{past and on-going research}, as well as \textbf{future research plan}.

\section{Statistical Methodology}
I am particularly interested in data integration, modeling and optimization for high-dimensional data, Bayesian methods, graphical models and statistical computing.
Note that some of these areas can potentially overlap.
I am not restricted to these areas I have explored.
If I encounter other meaningful and challenging problems,
I am definitely eager to learn or collaborate with other researchers.

\subsection{Data integration}

\begin{itemize}
\item \textbf{Past and on-going research:}

Due to rapid development of high-throughput experimental techniques and dropping prices, many transcriptomic datasets have
been generated and accumulated in the public domain (e.g. TCGA, GEO, SRA).
Single cohort/data type may suffer from small sample size issue.
A natural question is how to combine these complex data and increase statistical power, interpretation and reproducibility.
This includes two directions: \textbf{horizontal meta-analysis} and \textbf{vertical integrative analysis}. 
Horizontal meta-analysis aims to combine genomic data of same type from multiple cohorts.
I have worked on several meta-analytical methodologies including 
\textbf{disease subtype discovery}\cite{ref:MSKM}, 
\textbf{candidate marker detection}\cite{ref:BayesMP},
\textbf{differential co-expression network detection}\cite{ref:metaDCN}
and \textbf{dimensional reduction}\cite{ref:metaPCA}.
Vertical integrative analysis combines multi-omics data 
(e.g. gene expression, CNV, genotyping, methylation) of the same cohort.
I have developed a disease subtype discovery algorithm \textbf{integrating multi-level omics data with prior biological information}\cite{ref:ISKmeans}.
These methods will help better characterize a complex disease and develop towards personalized medicine.

\item \textbf{Future direction:}
\begin{enumerate}
\item I have worked on horizontal meta-analysis and vertical integrative analysis respectively. 
A natural extension is towards \textbf{two-way integration} by combining horizontal meta-analysis and vertical integrative analysis. 
\item 
More and more \textbf{epigenetic, single cell} and \textbf{neuroimaging data} are becoming available to help understand diseases with new insight.
Integrating these data with genomic data is a potential future direction.
\end{enumerate}

\end{itemize}





\subsection{Modeling and optimization for high dimensional data}
\begin{itemize}
\item \textbf{Past and on-going research:}

High-throughput data (e.g. genomic data) has more than 20,000 genes in human being but only tens or hundreds of samples.
This large $p$ and small $n$ problem brings statistical challenges to reveal disease mechanism behind the big data.
I am particularly interested in modeling high-dimensional data and solving related optimization problem with various forms of \textbf{regularizations}.
I have used a \textbf{lasso penalty} on clustering problems to formalize a statistical objective and perform optimization\cite{ref:MSKM}.
In another clustering problem, which required incorporating prior knowledge, 
I proposed \textbf{sparse overlapping group lasso} and used \textbf{alternating direction method of multipliers (ADMM)} to solve the challenging optimization problem\cite{ref:ISKmeans}.
By these models and techniques, we can discover the intrinsic information behind the high dimensional data.

\item \textbf{Future direction:}
\begin{enumerate}
\item I have worked on lasso, group lasso and overlapping group lasso problems.
Other regularization techniques such as \textbf{penalization on precision matrix} or \textbf{low rank penalty} are also appealing to genomic applications.
\item High dimensional problems are challenging from perspectives of both optimization and theory.
I have worked extensively on high-dimensional optimization problems.
\textbf{High-dimensional theory} is also interesting and challenging to me, which is essential for a  sound methodology with theoretical guarantees. 
\end{enumerate}

\end{itemize}

\subsection{Bayesian methods and graphical models}
\begin{itemize}
\item \textbf{Past and on-going research:}

Bayesian methods and graphical models are very flexible to model and reflect the biological generative process and its complex dependent structure.
I have worked on a \textbf{Bayesian non-parametric} approach to combine summary statistics from multiple cohorts to perform meta-analysis\cite{ref:BayesMP}.
I am working on \textbf{Bayesian variable selection} problems\cite{ref:MOG} with prior knowledge of multi-layer overlapping groups.
Bayesian approaches and graphical models are also growing fields themselves and I expect these techniques will play an important role in genomics and bioinformatics.

\item \textbf{Future direction:}
\begin{enumerate}
\item I have produced methodologies on high-dimensional clustering problem using frequentist approaches.
I am very interested in high-dimensional or clustering problems from the Bayesian perspective.
\item Part of my thesis is about methods for single cell data.
I proposed a \textbf{conditional random field} model for fast single cell imputation
and I'll continue exploring that.

\end{enumerate}

\end{itemize}


\section{Bioinformatics Application}
As a biostatistician,
an important job and privilege is to work with local biologists and play with their own data.
This is exciting for me because I can help biologists towards innovating scientific findings,
and their data can motivate me to develop relevant statistical methodology as well.

\begin{itemize}
\item \textbf{Past and on-going research:}

I have been mainly involved in \textbf{cancer} and \textbf{psychiatry} diseases research.
For cancer research, I have worked on disease subtypes of breast cancer\cite{ref:ILC},
DNA methylation of parity-induced mice\cite{ref:mouseParity}, 
copy number variation of prostate cancer\cite{ref:prostate},
and fusion transcript discovery\cite{ref:fusionGene}.
For psychiatry diseases,
I have worked on schizophrenia, bipolar disorder and major depressive disorder in human pyramidal neurons\cite{ref:MO1} and parvalbumin neurons\cite{ref:PVL3}.
Currently, I am also working on other aspects of psychiatry (e.g. Induced pluripotent stem cell \cite{ref:iPSC}, sex related depression effect \cite{ref:SexBLA, ref:SexAcross} and circadian pattern).

\item \textbf{Future direction:}
\begin{enumerate}
\item Collaboration is always an essential part for statistician/biostatistician.
I will definitely seek for opportunities to work with local biologists in my future tenure-track research environment, 
which will in turn motivate statistical methodology development.
\item I have worked extensively on microarray data and sequencing data.
But I understand that this is a fast moving field.
I am ready to learn newly developed data types, as technique advances.
At this stage, I am exposed to single cell data (single cell methylation and expression),
which is part of my thesis.
\end{enumerate}
\end{itemize}



\begin{thebibliography}{9}
\bibitem{ref:MSKM} 
{\bf Zhiguang Huo}, Ying Ding, Silvia Liu, Steffi Oesterreich, and George Tseng. Meta-Analytic Framework for Sparse K-Means to Identify Disease Subtypes in Multiple Transcriptomic Studies. \emph{Journal of the American Statistical Association},  111, no. 513 (2016): 27-42.
 
 \bibitem{ref:BayesMP} 
{\bf Zhiguang Huo}, Chi Song, George C. Tseng. (2016)
Bayesian latent hierarchical model for transcriptomic meta-analysis to detect biomarkers with clustered meta-patterns of differential expression signals. Submitted to \emph{Annals of Applied Statistics} (under second round of review).

\bibitem{ref:ISKmeans} 
{\bf Zhiguang Huo}, George C. Tseng. (2016)
Integrative Sparse $K$-means for disease subtype discovery using multi-level omics data.
Submitted to \emph{Annals of Applied Statistics} (under second round of review).

\bibitem{ref:metaDCN} 
Zhu, Li, Ying Ding, Cho-Yi Chen, Lin Wang, {\bf Zhiguang Huo}, SungHwan Kim, Christos Sotiriou, Steffi Oesterreich, and George C. Tseng. "MetaDCN: meta-analysis framework for differential co-expression network detection with an application in breast cancer." \emph{Bioinformatics} (2016): btw788.

\bibitem{ref:metaPCA} 
SungHwan Kim, Dongwan Kang, {\bf Zhiguang Huo}, Yongseok
Park, George C. Tseng. (2016)
Meta-analytic principal component analysis.
Submitted.

\bibitem{ref:MOG} 
Li Zhu, {\bf Zhiguang Huo}, Tianzhou Ma and George Tseng. 
Bayesian indicator variable selection model with multi-layer overlapping groups.
(in preparation).

\bibitem{ref:fusionGene} 
Silvia Liu, Wei-Hsiang Tsai, Ying Ding, Rui Chen, Zhou Fang, {\bf Zhiguang Huo}, SungHwan Kim, Tianzhou Ma, Ting-Yu Chang, Nolan Michael Priedigkeit, Adrian V. Lee, Jianhua Luo, Hsei-Wei Wang, I-Fang Chung, George C. Tseng. (2015).
Comprehensive evaluation of fusion transcript detection algorithms and a meta-caller to combine top performing methods in paired-end RNA-seq data.
\emph{Nucleic Acids Research}, 10.1093/nar/gkv1234.
 
\bibitem{ref:mouseParity} 
 Tiffany A. Katz, Serena G. Liao, Vincent J. Palmieri, Robert K. Dearth, Thushangi Pathiraja, {\bf Zhiguang Huo}, Patricia Shaw, Sarah Small, Nancy E. Davidson, David G. Peters, George C. Tseng, Steffi Oesterreich, Adrian V. Lee. (2015) Targeted DNA Methylation Screen in the Mouse Mammary Genome Reveals a Parity-Induced Hypermethylation of IGF1R That Persists Long after Parturition. \emph{Cancer Prevention Research} 8, no. 10 (2015): 1000-1009.

\bibitem{ref:prostate} 
Yan P. Yu, Silvia Liu, {\bf Zhiguang Huo}, Amantha Martin, Joel B. Nelson, George C. Tseng and Jian-Hua Luo. (2015) Genomic copy number variations in the genomes of leukocytes predict prostate cancer clinical outcomes. \emph{PloS one}, 10(8):e0135982.

    
\bibitem{ref:MO1} 
Dominique Arion, {\bf Zhiguang Huo}, John F. Enwright, John P. Corradi, George Tseng and David A. Lewis.
Transcriptome alterations in prefrontal pyramidal neurons distinguish schizophrenia from bipolar and major depressive disorders.
Submitted to \emph{Biological Psychiatry}, (under second round of review).



\bibitem{ref:ILC} 
Oesterreich, S., Katz, T.A., Logan, G., Levine, K., Nagle, A., {\bf Huo, Z.}, Tseng, G.C., Rui, H., Lee, A.V. and Butler, L.M., 2016. Abstract PD2-08: Potential role of prolactin signaling in development and growth of the lobular subtype of breast cancer. \emph{Cancer Research}, 76(4 Supplement), pp.PD2-08.

\bibitem{ref:PVL3} 
John Enwright,  Dominique Arion, {\bf Zhiguang Huo}, George Tseng and David A. Lewis. 
(2017) Transcriptome alterations in layer 3 parvalbumin neurons in the dorsolateral prefrontal cortex in schizophrenia differ from those in layer 3 pyramidal cells.
(in preparation).

\bibitem{ref:iPSC} 
 Logan, R.W., Ozburn, A.R., {\bf Huo, Z.}, Zhu, X., Fitzgerald, E., Arey, R.N., Jarpe, M., Tseng, G. and McClung, C.A. (2017)
Valproic acid targets HDAC2 to normalize mania-like behaviors in mice. In preparation.


\bibitem{ref:SexBLA}  Rachel Puralewski, {\bf Zhiguang Huo}, George Tseng, Ryan Logan, Marianne Seney. (2017) Sex differences in basolateral amygdala transcriptome associated with effects of chronic stress. In preparation.

\bibitem{ref:SexAcross}  {\bf Zhiguang Huo}, Rachel Puralewski, George Tseng, Ryan Logan, Marianne Seney. (2017) Circuit-wide transcriptional profiling reveals sex differences in coherence patterns across brain regions. In preparation.


\end{thebibliography}

\end{document}

